packages/gridstack/src/components/ConfigurableDashboard/ConfigurableDashboard.spec.ts

```diff
@@ -27,7 +27,9 @@ describe('ConfigurableDashboard.vue', () => {
         availableWidgets: new Map(),
         dashboardConfig: [],
         dashboardConfigurationOptions: [],
-        selectedDashboardConfiguration: undefined
+        selectedDashboardConfiguration: undefined,
+        showSaveModal: false,
+        showModalError: false
```

Comment:
instead of introducing a boolean here, u can pass an error here. So if an error is defined the error is displayed. If the prop is undefined no error is displayed.

Reply:
changed to passing error

Reply:
Why `showModalError` bool still exists? I do not see the error here